### PR TITLE
Fix shard summary links on Windows

### PR DESCRIPTION
**Pull request by fweikert**

### CODE REVIEW

**Comment by fweikert** [comment]
The diff is a bit weird because I ran black on it.

File: buildkite/bazelci.py [hunk 1]
```diff
@@ -3895,13 +3912,20 @@ def download_bep(self, dest_dir: str) -> str:
...
         try:
             return download_file(url, job_dir, _TEST_BEP_FILE)
         except:
             # TODO: handle exception
             return None
 
+    @property
+    def is_windows(self) -> bool:
+        # Dirty hack
+        return "\\" in self.relative_bep_path
```
Comment (meisterT) [inline buildkite/bazelci.py:3927]: If you want to check whether this is running on windows use `os == 'nt'` instead.

Comment (fweikert) [reply buildkite/bazelci.py:3927]: We process all BEP files across all platforms in a single job running on Linux, so the current os won't help.

Comment (meisterT) [reply buildkite/bazelci.py:3927]: Well, it is dirty, but it is what it is.

**Review by meisterT — APPROVED** [review]

**Pull request merged**